packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,52 @@ class FancyFocus extends FocusManager {
-| loading       | boolean                                                 | true     | False if an item is in focus, true otherwise                                                                                 |
-| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | alias for `children`                                                                                                         |
-| wrapSelected  | boolean                                                 | false    | enables wrapping behavior, so `selectNext` selects the first item if the current item is the last on the list and vice versa |
+| name         | type            | default | description                                                        |
+| ------------ | --------------- | ------- | ------------------------------------------------------------------ |
+| direction    | 'column'\|'row' | false   | Determines the key handling for focus ('left/right' or 'up/down'). |
+| wrapSelected | boolean         | false   | Enables wrapping behavior for focus navigation.                    |
 
 ### Methods
 
-#### appendItems
+#### appendItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>, idx: number): void
 
 Appends items to child list and updates focus
 
 ##### Arguments
 
-| name  | type                                                    | required | default | description     |
-| ----- | ------------------------------------------------------- | -------- | ------- | --------------- |
-| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | items to append |
+| name  | type                                                    | required | default | description                   |
+| ----- | ------------------------------------------------------- | -------- | ------- | ----------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append   |
+| idx   | number                                                  | true     | []      | The index to insert the items |
 
-#### render
+#### prependItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>): void
+
+Prepends items to the child list and updates the focus
+
+| name  | type                                                    | required | default | description                 |
+| ----- | ------------------------------------------------------- | -------- | ------- | --------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append |
+
+#### removeItemAt(index: number): void
+
+Removes an item at a specific index from the child list and updates the focus
+
+#### render(): void
```

Comment:
Out of curiosity, `render` is a private method in `FocusManager` but it's documented here. Is this expected? My thought was only public methods are listed/described in the documentation. Is `render` a special case?

@erautenberg

Reply:
It looks like this needs to be fixed in the TS as well to be underscored then. I don't have a problem documenting it, but it should have the appropriate name.